package.json

```diff
@@ -51,8 +51,9 @@
     "build": "babel --ignore=__tests__ src --out-dir lib",
     "lint": "standard",
     "prepublish": "npm test",
-    "posttest": "NODE_ENV=production npm run build",
+    "pretest": "npm run build",
```

Comment:
@jprichardson If I add `write(0, 'asdf')` to the source code, it will catch it, regardless of whether `pretest` exists or not.

If I call `write(0, 'asdf')` in the test files, it won't catch it, regardless of whether `pretest` exists or not.

Therefore, I think `pretest` is unneeded. Clear enough?